.github/ISSUE_TEMPLATE/BUG-REPORT.yaml

```diff
@@ -0,0 +1,39 @@
+name: Bug Report
+description: File a Corso bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Corso Version?
+      description: What version of Corso are you running?
+      value: "Corso vX.X.X"
+    validations:
+      required: true
+  - type: textarea
+    id: os
+    attributes:
+      label: Where are you running Corso?
+      description: Include OS version (e.g., macOS 13.0.1, Windows 11 Pro) and Object Storage system being used.
+      placeholder: Also include addition system details if relevant (e.g., shell, antivirus, firewall/proxy used on the network, etc.)
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```

Comment:
Should we ask them to enable debug logs as well as graph api queries(might have to remove identifiable info from the logs though)?
/cc @ryanfkeepers

Reply:
Related: https://github.com/alcionai/corso/pull/1860

Reply:
I assume error logs are more valuable to us than debug logs.

Reply:
@ryanfkeepers Right now, I think *all* logs are going to be valuable. What are the instructions to run with all logging enabled? Ideally, add it to the docs as a PR or just add it here and I can put it in the right place and link from here in a follow-on PR.

Reply:
Sure, I'll add it to a follow-up PR.